src/style.css

```diff
@@ -1 +1,118 @@
-/* add styles here */
+/* #region paje */
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf), url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0 8px;
+  padding: 0;
+  font-family: avenir, sans-serif;
```

Comment:
Consider adding a fallback font for 'avenir' to ensure text is displayed correctly if the font fails to load. For example: 'font-family: avenir, sans-serif;'